src/app/components/fees-row/fees-row.tsx

```diff
@@ -78,6 +79,7 @@ export function FeesRow(props: FeeRowProps): JSX.Element {
 
   return (
     <FeesRowLayout
+      data-testid={SendCryptoAssetSelectors.FeesSelector}
```

Comment:
There is actually a `FeesSelector` enum you can use here bc this is a shared component outside of just the send form.